rust/operator-binary/Cargo.toml

```diff
@@ -27,6 +27,7 @@ serde.workspace = true
 snafu.workspace = true
 strum.workspace = true
 h2.workspace = true
+tracing = "0.1.40"
```

Comment:
Can we please place that in the root Cargo.toml and only pull it in here?

Reply:
91336aaef6c62cf6efbcecc2ffe18f136d9bb2f0